src/stacking-group.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import Avatar from './avatar';
+import Image from './image';
+import * as styles from './stacking-group.css';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import classNames from 'classnames';
+
+type dataArrayType = {
+    src?: string;
+    initials?: string;
+};
+
+type StackingGroupProps = {
+    size: 24 | 32 | 64;
+    data: dataArrayType[];
+    type: 'avatar' | 'image';
+    stacked?: boolean;
+};
+
+const StackingGroup: React.FC<StackingGroupProps> = ({size = 24, data, type, stacked = false}) => {
+    // Due to the fact that the image does not have a variant to show the surplus of the array,
+    // as in the case of the avatar, a pseudo-image had to be created in this component
+
+    const isInverse = useIsInverseVariant();
+    const backgroundColor = isInverse ? vars.colors.brandHigh : vars.colors.brandLow;
+    const textColor = isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand;
+
+    let maxItems = 0;
+    if (stacked) {
+        maxItems = 4;
+    } else {
+        maxItems = 3;
+    }
+
+    const plusCase = data.length - maxItems;
+
+    const verify = type === 'avatar';
```

Comment:
verify what?

Reply:
The Verify const was to verify which element would be mounted in the Stacking Group being Avatar or Image but now it has been renamed to styleCircle where its function is to determine if the element that shows the excess number of elements will be a circle or a square.